lua/default-config.lua

```diff
@@ -1000,6 +1000,26 @@ lvim.lang = {
       },
     },
   },
+  gdscript = {
+    formatter = {
+      exe = "",
+      args = {},
+      stdin = false,
+    },
+    linters = {},
+    lsp = {
+      provider = "gdscript",
+      setup = {
+        cmd = {
```

Comment:
It would be better if you don't specify this as it's already defaulted by [lspconfig](https://github.com/neovim/nvim-lspconfig/blob/master/CONFIG.md#gdscript)

Reply:
I would like to let the user override the binary for `nc` using `lvim.lang.gdscript.lsp.setup.cmd[1] = "ncat"` as I stated above, also the user may want to use another port or IP instead of the default `6008` and `localhost` E.g when using Lunarvim inside of WSL2, the LSP server(Godot Engine in this case) is not `localhost`.

Reply:
You are right, it's more of a problem on our side since we force the values to be duplicated